site/_includes/icons/100-active.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This does not show up properly for me in the sidebar:

<img width="87" alt="Screen Shot 2022-03-24 at 3 14 56 PM" src="https://user-images.githubusercontent.com/1749548/159993447-2c583b95-3e9a-4129-9ed7-a9e2546b8476.png">

When the SVG is inactive, I see nothing.

Reply:
> Re: images optimization - all images have to go through our uploader pipeline: https://developer.chrome.com/docs/handbook/how-to/add-media/ The uploader outputs html code to use image as a nunjocks shortcode ("{% Img ... %}") but you can also just use the image's URL from the generated output to embed an image in a different way.
> 
> You might need a @google account to use the uploader - in this case please work with Harleen to upload the images and get the urls for them.

Used image uploader.